app/src/main/java/org/mozilla/focus/fragment/HomeFragment.kt

```diff
@@ -55,7 +59,16 @@ class HomeFragment : Fragment() {
     }
 
     private fun initTiles() = with (tileContainer) {
-        adapter = HomeTileAdapter(onUrlEnteredListener)
+        var homeTiles = mutableListOf<HomeTile>()
+        val inputAsString = context.assets.open(HOME_TILES_JSON_PATH).bufferedReader().use { it.readText() }  // defaults to UTF-8
+        val jsonArray = JSONObject(inputAsString).getJSONArray(HOME_TILES_JSON_KEY)
+        for (i in 0..(jsonArray.length() - 1)) {
+            val url = jsonArray.getJSONObject(i).getJSONObject("tile_" + i.toString()).getString("url")
```

Comment:
Let's get rid of this "tile_" string concatenation, I think it's unnecessary and adds complexity - json arrays should be ordered, right?